### ISSUE TITLE
Create marketing cookies category

### ISSUE THREAD

**Issue by LaurenFBaily** [wellcomecollection/wellcomecollection.org#10868]
### What?

Create marketing category for cookies categorised as marketing/advertising purpose and control cookies.

### Why?

Allow users opt in or opt out to marketing cookies via the preferences centre and cookie consent banner.

### Background

[See epic on Notion](https://www.notion.so/wellcometrust/Cookie-consent-dedae08913da484aadf75c2e4b004ecf?pvs=4])

Marketing cookies identified are:

- **YSC** Cookie set by YouTube to store and track views on embedded YouTube videos (session)
- **_gac_** Set by Google. Contains information related to marketing campaigns of the user. These are shared with Google AdWords / Google Ads when the Google Ads and Google Analytics accounts are linked together.
- **__tld__** Set by Segment. Used to track visitors on multiple websites, in order to present relevant advertisement based on the visitor's preferences. 

Pixels to support re-targeting advertising will be added at a later point.

### Done when

- [ ] Opt-in toggle and text added to the cookies preference centre
- [ ] Marketing cookies are controlled and only placed when user opts in via cookie consent banner or preferences centre
- [ ] Marketing cookies are cleared if user updates their preferences
- [ ] Marketing cookie info text added to Cookie policy page and cookies added to list

**Comment by LaurenFBaily** [comment]
Preference centre copy:

### Cookies for communications and marketing

We will use these to measure how you are interacting with our marketing and advertising materials, and the effectiveness of our campaigns.

**Comment by rcantin-w** [comment]
> Marketing cookie info text added to Cookie policy page and cookies added to list

@LaurenFBaily for the copy are we going with what's [in here](https://wellcomecloud.sharepoint.com/:w:/s/Legal/EUVgdmoNbilPsuhmS0u3-ugB1G7yZfoi-0FZwR4qsdr9HQ?e=2u4voQ)?


![Image](https://github.com/wellcomecollection/wellcomecollection.org/assets/110461050/d1e39a10-bcba-48bc-81e4-423190c75ec5)

**Comment by LaurenFBaily** [comment]
Yes. It's not a perfect fit until we add the marketing pixels but those are coming and it's the copy that's been agreed. Thanks!

**Comment by rcantin-w** [comment]
The two checkboxes that are left to do match the ones in #10717 (that includes all cookies), so I'd suggest closing this ticket once changes to the copy are approved and deployed.
